--- cmake/Modules/DefineInstallationPaths.cmake.old	2009-10-11 21:22:43.000000000 +0400
+++ cmake/Modules/DefineInstallationPaths.cmake	2010-09-07 12:16:11.000000000 +0400
@@ -83,7 +83,7 @@
 
 
 SET(SYSCONF_INSTALL_DIR
-  "${EXEC_INSTALL_PREFIX}/etc"
+  "/etc/${APPLICATION_NAME}"
   CACHE PATH "The ${APPLICATION_NAME} sysconfig install dir (default prefix/etc)"
   FORCE
 )
@@ -99,7 +99,7 @@
 )
 
 SET(LOCALSTATE_INSTALL_DIR
-  "${CMAKE_INSTALL_PREFIX}/var"
+  "/var/lib/${APPLICATION_NAME}"
   CACHE PATH "The ${APPLICATION_NAME} local state install dir (default prefix/var)"
   FORCE
 )
